src/discord.ts

```diff
@@ -17,6 +24,58 @@ discord.on("ready", async () => {
   )) as TextChannel;
 });
 
+discord.on("messageReactionAdd", async (messageReaction: MessageReaction, user: User | PartialUser) => {
```

Comment:
There probably needs to be a check that the reaction is on a message we're expecting, otherwise this emote response on any message may cause this to happen.

Reply:
@SociableSteve You are absolutely right. This slipped through during testing and forgot about this later. Fixed it in 
85abf00.

Reply:
√

Reply:
You are absolutely right. This slipped through during testing and forgot about this later. Added in 85abf00.